### PR TITLE
Cloaken integration

### PR DESCRIPTION
**Pull request by Itay4**
## Status
Ready

## Required version of Demisto
Any

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation
- [x] Code Review

### CODE REVIEW

**Comment by Itay4** [comment]
@kirbles19 please review Integrations/Cloaken/Cloaken.yml and Integrations/Cloaken/Cloaken_description.md

**Comment by kirbles19** [comment]
.yml is edited.
.md doesn't contain any detailed instructions, should remove.

File: Integrations/Cloaken/Cloaken.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Cloaken
+  version: -1
+configuration:
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: Server URL (e.g., https://cloaken.cypherint.com)
```
Comment (yaakovi) [inline Integrations/Cloaken/Cloaken.yml]: why server url is after the credentials?
better to have it as first parameter.

File: Integrations/Cloaken/Cloaken.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+# The command demisto.command() holds the command sent from the user.
```
Comment (yaakovi) [inline Integrations/Cloaken/Cloaken.py]: remove this

File: Integrations/Cloaken/Cloaken.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  name: server_url
+  defaultvalue: "https://cloaken.cypherint.com"
+  required: true
+  type: 0
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: do_not_verify_ssl
```
Comment (yaakovi) [inline Integrations/Cloaken/Cloaken.yml]: should be `insecure` / `unsecure`.
this is required for the mocks to run properly.

File: Integrations/Cloaken/Cloaken.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      isArray: false
+      name: url
+      required: true
+      secret: false
+    deprecated: false
+    description: Unshortens a URL.
+    execution: false
+    name: cloaken-unshorten-url
+    outputs:
+    - contextPath: Cloaken.unshortened_url
```
Comment (yaakovi) [inline Integrations/Cloaken/Cloaken.yml]: context path not in standard.

**Pull request merged**